.NET/Microsoft.Recognizer.Text.TimeZone/English/TimeZoneParser.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Text.RegularExpressions;
+using Microsoft.Recognizers.Definitions.English;
+
+namespace Microsoft.Recognizers.Text.TimeZone.English
+{
+    class TimeZoneParser : IParser
+    {
+        public TimeZoneParser()
+        {
+
+        }
+        public int getMinutes(string matched)
```

Comment:
Code standard. Please add blank lines between methods and methods names should follow TitleCase.